modulefiles/gfsutils_gaeac6.intel.lua

```diff
@@ -0,0 +1,32 @@
+help([[
+  This module loads libraries required for building and running GFS UTILS 
+  on the NOAA RDHPC machine Gaea C6 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea C6 ]===])
+
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/gsi-addon/install/modulefiles/Core")
```

Comment:
please remove the duplicate line

Reply:
I notice its not duplicate.  Why are both unified-env and gsi-addon envs needed to be loaded.

Reply:
@aerorahul unified-env has bufr/12.0.1, and gsi-addon has bufr/11.7.0 needed by gfs-utils. Initially when I ported to C5, I had to have both MODULEPATHs. I just tested C5 with only gsi-addon, and the build was successful. I haven't had access to C6 today so will test that build when I can.

Reply:
gfs-utils has been tested on both C5 and C6.